src/js/modules/infragistics.ui.tree.js

```diff
@@ -3039,7 +3039,8 @@
 			}
 			var ul = this._populatingNode.ul, node = this._populatingNode.node,
 				originalData = this.nodeDataFor(node.attr("data-path")),
-				depth = ul.attr("data-depth"),
+				// I.G. January 31st, 2018 #1564 igTree does not load nodes after lv3 when using loadOnDemand
+				depth = parseInt(ul.attr("data-depth")),
```

Comment:
jshint.

Reply:
Canceling the pull request for 17.1 as the bug does not reproduce here and it was just nice to have it here.